addons/html_builder/static/src/snippets/snippet_viewer.xml

```diff
@@ -8,6 +8,7 @@
             <t t-foreach="snippetsColumn" t-as="snippet" t-key="snippet.id">
```

Comment:
you can add a `t-ref="content"` on the root div from this template (the one with o_snippets_preview_row)